exercises/concept/beers/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+You are the new bosun of a small pirate crew that just killed their previous bosun for fowling their merriment with a bad song. As pirates would have it, your crew likes to pass the time getting loaded to the gunwales while crossing the briny deep and you are in charge of rationing the beer. The Cap'n has tasked you to minimize the number of resupply trips to make the best possible time between hauls. To achieve these conflicting requirements, and keep your crew from blowing you down when you are also loaded to the gunwales, you decide to stick to the tried-and-true 99 Bottles of Beer song using a program to help you keep the song straight and remind you to make a beer run when the ship is running low.
```

Comment:
This in a very entertaining introduction to the exercise, and I thoroughly enjoyed reading it. I think it will probably make difficult reading for a non native english speaker though. Probably @mpizenberg and @jiegillet can comment on this, as they are non native english speakers.
Maybe we can find a way to keep the fun text but also make life easier for students. We can have a think, but maybe just a bit at the start stating that the following paragraph is not critical for the exercise.